docker/README.md

```diff
@@ -0,0 +1,105 @@
+# Running Kubernetes-Mesos In Docker
+
+There are several ways to play with kubernetes-mesos in docker. Here are a couple that might be most useful:
+
+1. [Complete](#complete)
+  - Run components and dependencies all in the same container
+  - All containers mirroring localhost ports (docker host mode)
+2. [Composed](#composed)
+  - Run components and dependencies each in different containers
+  - Each container gets its own IP
+  - Requires an ambassador to proxy ports (docker doesn't allow circular hostname linking)
+  - Emulates a more complex system, useful for testing networking
+
+While it is technically possible to run docker within a docker container, doing so requires running the mesos (slave) in priveledged mode.
+So the simplest way is to instead expose the bind mount the docker socket (`-v "/var/run/docker.sock:/var/run/docker.sock"`)
+
+In addition to running in docker, it's also possible to [run docker locally](#local).
+
+<a name="complete"/>
+## K8SM Complete
+
+The "complete" Dockerfile includes everything needed to run a development instance of kubernetes-mesos, including etcd and mesos.
+
+### Background mode
+
+```
+docker run --name kubernetes-mesos -p 8888:8888 -p 5050:5050 -p 4001:4001 mesosphere/kubernetes-mesos &> /tmp/kubernetes-mesos-docker.log &
```

Comment:
Usually one leaves out the &> ... redirection and just uses `docker logs` afterwards.